api_management/apps/analytics/csv_analytics/metrics_calculator.py

```diff
@@ -57,6 +57,9 @@ def calculate_indicators(self, queries, row_date):
         indicator_row.save()
 
     def calculate(self, force):
+        if Query.objects.filter(api_data__name=self.api_name).count() == 0:
```

Comment:
`if not Query.objects...`. Regla talibán de python

Reply:
`If not Query.objects.filter(api_data__name=self.api_name)` es la forma?

Reply:
con el count al final, sí